roles/pulp_devel/templates/alias.bashrc.j2

```diff
@@ -160,7 +160,7 @@ pbindings(){
         cd pulp-openapi-generator
     fi
 
-    if command podman 2> /dev/null; then
```

Comment:
not sure what happened, it wasn't working yesterday, but it worked today!

Reply:
You fixed the syntax though. `command -v` checks for the executable and prints its path to standard output, `command` runs the executable.